src/node/install.sh

```diff
@@ -370,20 +378,21 @@ if [ ! -z "${ADDITIONAL_VERSIONS}" ]; then
 fi
 
 # Install pnpm
-if bash -c ". '${NVM_DIR}/nvm.sh' && type pnpm >/dev/null 2>&1"; then
-    echo "pnpm already installed."
+if bash -c ". '${NVM_DIR}/nvm.sh' && type npm >/dev/null 2>&1"; then
+    (
+        . "${NVM_DIR}/nvm.sh"
+        [ ! -z "$http_proxy" ] && npm set proxy="$http_proxy"
+        [ ! -z "$https_proxy" ] && npm set https-proxy="$https_proxy"
+        [ ! -z "$no_proxy" ] && npm set noproxy="$no_proxy"
+        PNPM_VERSION="pnpm"
+        if [ "${NODE_VERSION}" = "16" ]; then
```

Comment:
Rethinking about this, in future it will end up piling too many conditions.

Is there a dynamic way to find out the compatibility between node versions and pnpm?

Reply:
removed